trigger_jenkins_job.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
-from __future__ import print_function
+#!/usr/bin/env python3
```

Comment:
were you able to trigger any jenkins job using this?

Reply:
looks fine, I was able to trigger https://cmssdt.cern.ch/jenkins/view/Jenkins%20Tests/job/jenkins-test-job3/1/console